css/wo-ist-markt.css

```diff
@@ -0,0 +1,58 @@
+@font-face {
+  font-family: 'wo-ist-markt';
+  src: url('../font/wo-ist-markt.eot?25033009');
+  src: url('../font/wo-ist-markt.eot?25033009#iefix') format('embedded-opentype'),
+       url('../font/wo-ist-markt.woff2?25033009') format('woff2'),
+       url('../font/wo-ist-markt.woff?25033009') format('woff'),
```

Comment:
Can you document in your commit message details what the individual file format are for? You can update your commit like this:

``` bash
git commit --amend
// update commit message
git push --force origin basket-icon
```

What is the `?25033009` for?

Reply:
Thanks. Found it in the commit message.